packages/hoppscotch-backend/src/user/user.service.spec.ts

```diff
@@ -414,6 +434,41 @@ describe('UserService', () => {
     });
   });
 
+  describe('updateUser', () => {
+    test('should resolve right and update user', async () => {
+      mockPrisma.user.update.mockResolvedValueOnce(user);
+
+      const result = await userService.updateUser(user.uid, user.displayName);
+      expect(result).toEqualRight({
+        ...user,
+        currentGQLSession: JSON.stringify(user.currentGQLSession),
+        currentRESTSession: JSON.stringify(user.currentRESTSession),
+      });
+    });
```

Comment:
This test is technically not invalid, but I think we should actually write the test to update to a different name and check.